subsys/bluetooth/host/classic/hfp_ag.c

```diff
@@ -1601,6 +1601,134 @@ static int bt_hfp_ag_nrec_handler(struct bt_hfp_ag *ag, struct net_buf *buf)
 	return -ENOTSUP;
 }
 
+static void btrh_held_call_updated_cb(struct bt_hfp_ag *ag, void *user_data)
+{
+	bt_hfp_ag_set_call_state(ag, BT_HFP_CALL_ACTIVE);
+}
+
+static void btrh_held_call_setup_updated_cb(struct bt_hfp_ag *ag, void *user_data)
+{
+	if (bt_ag && bt_ag->incoming_held) {
+		bt_ag->incoming_held(ag);
+	}
+}
+
+static void btrh_held_cb(struct bt_hfp_ag *ag, void *user_data)
+{
+	int err;
+
+	err = hfp_ag_update_indicator(ag, BT_HFP_AG_CALL_IND, 1, btrh_held_call_updated_cb, NULL);
+	if (err != 0) {
+		LOG_ERR("Fail to send err :(%d)", err);
+		bt_hfp_ag_unit_call_terminate(ag, user_data);
+	}
+
+	err = hfp_ag_update_indicator(ag, BT_HFP_AG_CALL_SETUP_IND, BT_HFP_CALL_SETUP_NONE,
+					btrh_held_call_setup_updated_cb, NULL);
+	if (err != 0) {
+		LOG_ERR("Fail to send err :(%d)", err);
+		bt_hfp_ag_unit_call_terminate(ag, user_data);
+	}
+}
+
+static void btrh_accept_cb(struct bt_hfp_ag *ag, void *user_data)
+{
+	int err;
+
+	if (bt_ag && bt_ag->accept) {
+		bt_ag->accept(ag);
+	}
+
+	err = bt_hfp_ag_create_audio_connection(ag);
+	if (err) {
```

Comment:
And here just `(err)`. Please just pick one style.

Reply:
Updated.